app/build.gradle

```diff
@@ -22,9 +22,12 @@ android {
 
     lintOptions {
         warning 'MissingTranslation'
-        warning 'InvalidPackage' // https://github.com/square/okio/issues/58
+        disable 'InvalidPackage' // http://jakewharton.github.io/butterknife/
```

Comment:
link? But a) let's not disable this check - let's just not let it break
the build b) I think the url before was more helpful than the new one

On 06.10.2015 14:18, Tobias Preuss wrote:
>  disable 'InvalidPackage'


-- 
best regards,
ligi
http://ligi.de